SkyEditor.ROMEditor.Windows/Windows/Java.vb

```diff
@@ -1,15 +1,30 @@
-﻿Namespace Windows
+﻿Imports System.IO
+Imports System.Windows.Forms
+
+Namespace Windows
     Public Class Java
 
         Public Shared Async Function RunJar(JarPath As String, Arguments As String, WorkingDirectory As String) As Task
             Dim args As New Text.StringBuilder
-            args.Append("-jar ")
-            args.Append($"""{JarPath}""")
+
+            Try
+                args.Append("-jar ")
+                args.Append($"""{JarPath}""")
+                Console.WriteLine("Try has Finished")
+            Catch ex As FileNotFoundException
+                MessageBox.Show("Java Not Found: " & ex.Message)
```

Comment:
In a normal application that does simple things, this would be a reasonable solution. However, let's consider the context in which this function is called. [Here's the call to unluac](https://github.com/evandixon/SkyEditor.ROMEditor/blob/master/SkyEditor.ROMEditor.Windows/MysteryDungeon/PSMD/Projects/PsmdLuaProject.vb#L266). That function is called once per compiled LUA script in the game, resulting in quite a lot of these message boxes being shown. Luckily I have some logic to limit how many will be shown at once (otherwise there'll be hundreds of instances of Java running, which will make any computer slow to a halt). Once the OK button is clicked, however, everything will continue as normal, since there's no exception being thrown, so those message boxes will keep popping up.

A better solution would be to throw a brand-new exception, one that will bubble up, stop everything, and give a user-friendly indication that Java isn't installed, as is now done in the lastest commit: 534f29d02e8c9c2cb92cbbc9c512b09184c5c985